keras2onnx/ke2onnx/main.py

```diff
@@ -91,23 +91,26 @@ def convert_keras_concat(scope, operator, container):
 
 def convert_keras_flatten(scope, operator, container):
     iop = operator.raw_operator
-    target_shape = 1
-    for idx, val in enumerate(iop.output_shape):
-        if idx > 0:
-            target_shape = target_shape * val
-    target_shape = (-1, target_shape)
     shape_len = len(iop.input_shape)
+
     if iop.data_format == 'channels_last' or shape_len < 3:
-        apply_reshape(scope, operator.inputs[0].full_name, operator.outputs[0].full_name, container,
-                      operator_name=operator.raw_operator.name, desired_shape=target_shape)
+        _apply_flatten(scope, operator.inputs[0].full_name, operator.outputs[0].full_name, container,
+                       operator_name=operator.raw_operator.name)
     else:
         perm = list(range(2, shape_len))
         perm = [0] + perm + [1]
         input_tensor_name = scope.get_unique_variable_name(operator.inputs[0].full_name + '_permuted')
         apply_transpose(scope, operator.inputs[0].full_name, input_tensor_name, container,
                         operator_name=operator.raw_operator.name + "_transpose", perm=perm)
-        apply_reshape(scope, input_tensor_name, operator.outputs[0].full_name, container,
-                      operator_name=operator.raw_operator.name, desired_shape=target_shape)
+        _apply_flatten(scope, input_tensor_name, operator.outputs[0].full_name, container,
+                       operator_name=operator.raw_operator.name)
+
+
+# TODO: This function should be moved to onnxconverter_common.onnx_ops
+def _apply_flatten(scope, input_name, output_name, container, operator_name=None):
+    from onnxconverter_common.onnx_ops import _create_name_or_use_existing_one
```

Comment:
would you please add apply_flatten in  onnxconverter-common before this PR?

Reply:
This PR will take care of it: https://github.com/microsoft/onnxconverter-common/pull/31

Reply:
just merged this change, thanks

Reply:
@jiafatom, thank you!

@wenbingl, do we have to wait until this change makes it into a release?

Reply:
All keras-onnx CI build and tests are depended on the latest master branch of onnxconverter-common, so there is no wait. And we will release onnnxconverter-common before the next release of keras-onnx.

Reply:
Done.